### ISSUE TITLE
Multiline Comments Only Showing Last 4 Lines 

### ISSUE THREAD

**Issue by anguspmitchell** [TeamCodeStream/codestream#709]
**Expected**: PR comment on entire selection (11 lines)
**Actual**: PR comment only covers last 4 lines
**Versions**:
- VSCode 1.58.2
- OS: Darwin x64 18.7.0
- Node.js: 14.16.0
- Codestream: 11.0.3

Creating Comment:
![image](https://user-images.githubusercontent.com/4749149/132561238-32b2684b-344f-49a0-8d33-09ed0e845214.png)

What it looks like on GitHub after submitting
![image](https://user-images.githubusercontent.com/4749149/132561358-d188a8d3-646a-4a69-964c-eaf674622ee4.png)

What it looks like in VSCode after submitting:
![image](https://user-images.githubusercontent.com/4749149/132561396-c583b28b-1cfc-47d8-9047-a6e08d3c94cf.png)

**Comment by planteater** [comment]
@anguspmitchell - In the report you indicated that you're using v11.0.3 of the CodeStream extension. Was that just a typo? If not, could I trouble you to try again with the latest verion, which is 11.0.14. Thanks.

**Comment by anguspmitchell** [comment]
Just confirmed that problem persists with v11.0.15. VSCode 1.60.1. @planteater I'll send you the logs in an email.

```
-> node -v
v16.7.0
```

```
-> nvm list
         v5.0.0
         v7.0.0
         v9.4.0
       v11.14.0
       v14.17.0
->      v16.7.0
default -> node (-> v16.7.0)
node -> stable (-> v16.7.0) (default)
stable -> 16.7 (-> v16.7.0) (default)
iojs -> N/A (default)
lts/* -> lts/fermium (-> N/A)
lts/argon -> v4.9.1 (-> N/A)
lts/boron -> v6.17.1 (-> N/A)
lts/carbon -> v8.17.0 (-> N/A)
lts/dubnium -> v10.24.1 (-> N/A)
lts/erbium -> v12.22.5 (-> N/A)
lts/fermium -> v14.17.5 (-> N/A)
```

**Comment by planteater** [comment]
Fixed in v12.0.3.

**Comment by anguspmitchell** [comment]
Thank you!
